src/components/TextEditor.vue

```diff
@@ -380,6 +380,9 @@ https://github.com/ckeditor/ckeditor5/issues/1142
  .ck .ck-reset {
 	background: var(--color-main-background) !important;
  }
+.ck-content ul, .ck-content ol {
+	padding-inline-start: 40px;
```

Comment:
Where does the magic number come from? Is that the CKEditor default?

Reply:
> Where does the magic number come from? Is that the CKEditor default?

yes

Reply:
then leave this as code comment please